src/symbols/Label.tsx

```diff
@@ -1,7 +1,49 @@
 import React, { FC, useMemo } from 'react';
-import { Billboard, Text } from 'glodrei';
+import { Billboard, RoundedBox, Text } from 'glodrei';
 import { Color, ColorRepresentation, Euler } from 'three';
 import ellipsize from 'ellipsize';
+import { a } from '@react-spring/three';
+
+const calculateTextSize = (
```

Comment:
We have something similar to this in reaviz - https://github.com/reaviz/reaviz/blob/master/src/common/utils/size.ts#L8